### PR TITLE
MB-50049 Release note for 6.6.4 minor release

### PR DESCRIPTION
**Pull request by RayOffiah**
Bug fixes, including upgrade of Log4J to fix vulnerability

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,6 +5,40 @@
...
+
+[#table_fixedissues_v664-analytics,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-50049[MB-50049^]
+| *Summary*: Update Log4J to 2.15. 
+Fixes https://nvd.nist.gov/vuln/detail/CVE-2021-44228[CVE-2021-44228^]
+
+| https://issues.couchbase.com/browse/MB-49022[MB-49022^]
```
Comment (RichardSmedley) [inline modules/release-notes/pages/relnotes.adoc]: afaict the 4 issues not Log4J will be in 6.6.5, not 6.6.4

**Review by RichardSmedley — APPROVED** [review]

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,6 +5,42 @@
...
+
+[#fixed-issues-664]
+=== Fixed issues
+
+This section highlights the notable issues fixed in this release.
+
+[IMPORTANT]
+====
+This release fixes a high severity vulnerability related to Log4j 2. 
+All users should upgrade to this release as soon as possible.
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc]: Can we change this to
All users utilising the Couchbase Analytics service should upgrade to this release as soon as possible.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,42 @@
...
+_Quick Links_: <<fixed-issues-664>> | <<common-vulnerabilities-exposures-664>>
+
+[#fixed-issues-664]
+=== Fixed issues
+
+This section highlights the notable issues fixed in this release.
+
+[IMPORTANT]
+====
+This release fixes a high severity vulnerability related to Log4j 2.
```
Comment (ibsoln) [inline modules/release-notes/pages/relnotes.adoc:23]: Is that the formal name Log4j 2. ?

File: modules/release-notes/pages/relnotes.adoc [hunk 4]
```diff
@@ -5,6 +5,42 @@
...
+
+_Quick Links_: <<fixed-issues-664>> | <<common-vulnerabilities-exposures-664>>
+
+[#fixed-issues-664]
+=== Fixed issues
+
+This section highlights the notable issues fixed in this release.
+
+[IMPORTANT]
+====
```
Comment (ibsoln) [inline modules/release-notes/pages/relnotes.adoc:22]: Interesting. I normally just use a `--` block for these.

**Review by ibsoln — APPROVED** [review]
Checked links and text. Minor cosmetic which may well be right anyhow. So seems good to go.

**Pull request merged**